libs/expo/betterangels/src/lib/screens/AddEditClient/HouseholdMembers/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { Colors, Spacings } from '@monorepo/expo/shared/static';
+import { Accordion, TextButton } from '@monorepo/expo/shared/ui-components';
+import { RefObject } from 'react';
+import { useFieldArray, useFormContext } from 'react-hook-form';
+import { ScrollView, View } from 'react-native';
+import {
+  CreateClientProfileInput,
+  UpdateClientProfileInput,
+} from '../../../apollo';
+import Household from './Household';
+
+interface IHouseholdMembersProps {
+  expanded: undefined | string | null;
+  setExpanded: (expanded: undefined | string | null) => void;
+  scrollRef: RefObject<ScrollView>;
+}
+
+export default function HouseholdMembers(props: IHouseholdMembersProps) {
+  const { scrollRef, expanded, setExpanded } = props;
+
+  const { control } = useFormContext<
+    UpdateClientProfileInput | CreateClientProfileInput
+  >();
+
+  const { fields, append, remove } = useFieldArray({
+    control,
+    name: 'householdMembers',
+  });
+
+  const isHouseholdMembers = expanded === 'Household';
+
+  return (
+    <Accordion
```

Comment:
when the dropdown is close to the bottom of the screen, and there's no more room to scroll, it's impossible to access the bottom items in the dropdown

<img width="301" alt="image" src="https://github.com/user-attachments/assets/21cd1a0d-5fe2-409e-bd22-6e151a84f425">

Reply:
knew this. we might change dropdown library